test/meson.build

```diff
@@ -24,13 +24,9 @@ test_utils_files = [
     'options/m_config_core.c',
```

Comment:
Can't comment where I want because github is stupid, but the comment above this should be corrected. Probably just remove the `Paths and subprocesses...` sentence.

Reply:
done